test/unit/security_test.py

```diff
@@ -0,0 +1,360 @@
+import json
+import os
+import subprocess
+from contextlib import contextmanager
+from inspect import cleandoc
+from unittest import mock
+
+import pytest
+
+from exasol.toolbox.tools import security
+
+
+@contextmanager
+def empty_path():
+    """Make sure the PATH environment variable is empty."""
+    old_path = os.environ["PATH"]
+    os.environ["PATH"] = ""
+    yield
+    os.environ["PATH"] = old_path
+
+
+class TestCreateSecurityIssue:
+    @pytest.mark.parametrize(
+        "expected,issue",
+        [
+            (
+                "🔐 CVE-2023-39410: pkg:maven/fr.turri/aXMLRPC@1.13.0",
```

Comment:
be careful with unicode signs in tickets, we saw already some issues with that in the past

Reply:
* As discussed I'll talk with @redcatbear if this is a no go